### ISSUE TITLE
[A11Y] Dashboard reflow issues at 200–400% zoom

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#12559]
## Issue Description
Users who magnify or use small screens cannot access core workflows when layout does not reflow to a single column at ~320 CSS px.  Main content is clipped/off-screen; at 400% only the left nav is reachable and critical views are unusable.

## How to reproduce / Current behavior
Using a browser desktop:
1. Log in to the platform
2. Increase browser zoom to 200% then 400%.
3. Attempt to access main content without horizontal scrolling or endless vertical scrolling.
4. Using keyboard only , attempt to navigate through interface

## What application are you using?
 OJS, OMP or OPS version 3.5.0-3

## Stack used 
- Device: Laptop, mobile
- OS: MacOS, Windows
- Browser: Firefox, Chrome

## Testing method
- Method:  Manual; Keyboard; screenreader

## Action
Review the components in the interface to reflow and fit to  400% zoom (1280→320 CSS px) allowing vertical scrolling.

## WCAG Success Criteria
1.4.10 Reflow (AA)

## Additional information

**Zoom 200%**
<img width="212" height="104" alt="Image" src="https://github.com/user-attachments/assets/0e9cf5c2-8999-43c2-9f68-b3ff67583e9f" />

**Zoom 400%**

<img width="212" height="106" alt="Image" src="https://github.com/user-attachments/assets/37328f22-ef80-4807-956b-65447b17a845" />
